### ISSUE TITLE
ModuleNotFoundError: No module named 'openlocationcode'

### ISSUE THREAD

**Issue by thierry-FreeBSD** [Kosmorro/kosmorro#235]
### I have searched for an opened issue and didn't find my bug

- [X] I confirm I didn't find any issue regarding my bug

### Version of Kosmorro

0.10

### Version of Python

3.8

### Operating system

FreeBSD

### Installation source

I have compiled the package from source

### Bug description

Launching kosmorro 0.10.8 produces the following error:
```
Traceback (most recent call last):
  File "/usr/local/bin/kosmorro", line 20, in <module>
    from _kosmorro.main import main
  File "/usr/local/lib/python3.8/site-packages/_kosmorro/main.py", line 31, in <module>
    from .geolocation import get_position
  File "/usr/local/lib/python3.8/site-packages/_kosmorro/geolocation.py", line 7, in <module>
    from openlocationcode import openlocationcode
ModuleNotFoundError: No module named 'openlocationcode'
```

**Comment by Deuchnord** [comment]
Are you sure you are on the right tag/branch?
The trace seems to correspond to the `feature` branch, not to the `0.10.8` tag

**Comment by thierry-FreeBSD** [comment]
I just downloaded the tarball kosmorro-0.10.8.tar.gz, from https://pypi.org/project/kosmorro/#files

**Comment by Deuchnord** [comment]
OK, just found what happened. Looks like I accidentally pushed the next (unfinished) major version (1.0.0).
I'm making a new release to fix that, thanks for the report :)

**Comment by Deuchnord** [comment]
Repoening this until the v0.10.9 is published on PyPI, there are multiple issues to fix to unblock it.

**Comment by Deuchnord** [comment]
Just published version 0.10.9 that should fix the issue. Thanks for your report, Thierry!
Feel free to reopen this issue if something is still wrong :)

**Comment by Deuchnord** [comment]
@all-contributors add @thierry-FreeBSD for packaging and bug report
